python/whylogs/core/schema.py

```diff
@@ -22,6 +22,15 @@
 T = TypeVar("T", bound="DatasetSchema")
 
 
+@dataclass
+class AugmenterSpec:
+    column_name: Optional[str] = None  # TODO: maybe make this a regex
+    column_type: Optional[Any] = None
+    callbacks: Dict[str, Callable[[Any], Any]] = field(
```

Comment:
maybe `callbacks: List[Callable[[Any], Tuple[str, Any]]]` where the returned string is the new column name.